tests/shell/external/example_codegen_econf_empty_array_spec.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+
+@INCLUDE_COMMON@
+
+echo
+echo ELEKTRA CHECK EXTERNAL CODEGEN ECONF EMPTY ARRAY SPEC
+echo
+
+if command -v realpath; then
+	if command -v pkg-config; then
+		if ! pkg-config elektra; then
+			echo "Elektra not installed, will skip"
+			exit 0
+		fi
+	else
+		echo "pkg-config not installed, will skip"
+		exit 0
+	fi
+else
+	echo "realpath is not installed, will skip"
+	exit 0
+fi
+
+check_version
+
+EXTERNAL_FOLDER="@CMAKE_SOURCE_DIR@/examples/codegen/econf"
+
+check_if_parent_exists() {
+	if grep -e "spec:\/sw\/example\/econf/\#0\/current\/format\/#$"; then
+		echo "array parent was found"
+	else
+		echo "did not find array parent spec:/sw/example/econf/#0/current/format/#"
```

Comment:
Just to be clear: Do you actually fail the test, or do you just print this message?

Reply:
good catch, I almost forgot to add it to the else case.